tests/test_pydantic.py

```diff
@@ -5,16 +5,35 @@
 import ormsgpack
 
 
-class Model1(BaseModel):
-    hi: str
-    number: int
-    sub: Optional[int]
+def test_pydantic_model() -> None:
+    class Model(BaseModel):
+        a: str
+        b: int
+        c: Optional[int]
 
-
-def test_basemodel() -> None:
-    """
-    packb() pydantic basemodel
-    """
-    obj = Model1(hi="a", number=1, sub=None)
+    obj = Model(a="a", b=1, c=None)
     packed = ormsgpack.packb(obj, option=ormsgpack.OPT_SERIALIZE_PYDANTIC)
-    assert ormsgpack.unpackb(packed) == {"hi": "a", "number": 1, "sub": None}
+    assert ormsgpack.unpackb(packed) == {
+        "a": "a",
+        "b": 1,
+        "c": None,
+    }
+
+
+def test_pydantic_model_sort_keys() -> None:
+    class Model(BaseModel):
+        b: int
+        c: Optional[int]
+        a: str
+
+    obj = Model(a="a", b=1, c=None)
+    packed = ormsgpack.packb(
+        obj,
+        option=ormsgpack.OPT_SERIALIZE_PYDANTIC | ormsgpack.OPT_SORT_KEYS,
+    )
+    assert list(obj.__dict__.keys()) != sorted(obj.__dict__.keys())
+    assert list(ormsgpack.unpackb(packed).items()) == [
+        ("a", "a"),
+        ("b", 1),
+        ("c", None),
+    ]
```

Comment:
I'd add an extra assertion, (maybe superfluous) where I check that `ormsgpack.packb`-ed version of the instance converted to a dictionary yields the same bytes.
```suggestion
    ]
    packed_dict = packed = ormsgpack.packb({"a", b: 1.; c: None}, option=ormsgpack.OPT_SORT_KEYS)
    # packed_dict = packed = ormsgpack.packb(obj.model_dump(), option=ormsgpack.OPT_SORT_KEYS)
    assert packed = packed_dict
```